storage/libradb/src/schema/transaction_info/test.rs

```diff
@@ -2,18 +2,13 @@
 // SPDX-License-Identifier: Apache-2.0
```

Comment:
Does this test provide any practical value at this point in time? I've been contemplating what is the value of these bidirectional tests if everything is machine generated.

Reply:
In this case it's probably not very useful since the encoding just wraps LCS. For some of the stuff where encoding is manually done, it's useful to test the encode/decode roundtrip.